lib/api/apiUtils/object/abortMultipartUpload.js

```diff
@@ -161,9 +163,13 @@ function abortMultipartUpload(authInfo, bucketName, objectKey, uploadId, log,
                 uploadId,
                 versionId: objectMD.versionId,
             });
-            return metadata.deleteObjectMD(bucketName, objectKey, {
-                versionId: objectMD.versionId,
-            }, log, err => {
+
+            const options = preprocessingVersioningDelete(
+                bucketName, destBucket, objectMD, objectMD.versionId, config.nullVersionCompatMode);
+            options.replayId = uploadId;
+            options.doesNotNeedOpogUpdate = true;
```

Comment:
Can we add a comment, just explaining it means in this cleanup case, this is a ghost (md, but no data, yet not a data loss), so we do not want to affect the backend services with useless oplog events, hence this option set to true?